### PR TITLE
AP-1018 add new schema for geofencing

### PR DESCRIPTION
**Pull request by dzhang30**
schema_validation for geofencing module on datapush
test this with https://github.com/humanprojectinc/control_server/pull/686

### CODE REVIEW

File: docs/android/geofencing/002.json [hunk 1]
```diff
@@ -1,49 +1,44 @@
...
           },
-          "eventType": {
-            "type": "string"
-          },
-          "geofenceRadius": {
+          "radius": {
               "type": "number",
               "minimum": 0
+          },
+          "snoozed": {
```
Comment (manvitadalal) [inline docs/android/geofencing/002.json:38]: Let's make 'snoozed' optional so our solution is expandable for other geofence use cases as well.

File: docs/ios/geofencing/002.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          "longitude": {
+              "type": "number",
+              "minimum": -180,
+              "maximum": 180
+          },
+          "radius": {
+              "type": "number",
+              "minimum": 0
+          },
+          "snoozed": {
```
Comment (manvitadalal) [inline docs/ios/geofencing/002.json:38]: Let's make 'snoozed' optional so our solution is expandable for other geofence use cases as well.

**Review by manvitadalal — CHANGES_REQUESTED** [review]
LG overall. Just 1 small change so snooze can be an optional feature in the future without having to change the schema.

File: docs/ios/geofencing/002.json (hunk 2 above)
Comment (dzhang30) [reply docs/ios/geofencing/002.json:38]: check line 42 now

File: docs/android/geofencing/002.json (hunk 1 above)
Comment (dzhang30) [reply docs/android/geofencing/002.json:38]: check line 42 now

**Review by manvitadalal — APPROVED** [review]
LG

**Review by kailashrkk — APPROVED** [review]
lgtm

**Pull request merged**